ibis/expr/operations/analytic.py

```diff
@@ -237,4 +231,16 @@ class NthValue(Analytic):
     output_dtype = rlz.dtype_like("arg")
 
 
+# TODO(kszucs): should inherit from analytic base
+@public
+class TopK(Node):
```

Comment:
Ugh, hopefully we can get rid of this in a follow-up unless it's still somehow required or useful.